src/components/MainSection.jsx

```diff
@@ -1,36 +1,45 @@
 import React from "react";
 import Wrapper from "./Wrapper";
-import Arrow from "/src/img/arrow.png";
+import CompassArrow from "/src/img/compass-arrow.svg";
+import CompassCircle from "/src/img/compass-circle.svg";
 import CircleButton from "/src/img/circle-button.svg";
 
 function MainSection() {
   return (
     <main>
       <section className="bg-[url('/src/img/main-section-bg.jpg')] bg-cover bg-center w-full h-screen border-y border-secondary-color">
         <Wrapper>
-          <div className="relative h-screen flex flex-col md:flex-row justify-center items-center">
-            <div className="md:grow text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark">
+          <div className='relative h-screen flex justify-center items-center'>
+            <div className='text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark'>
               <p>Lorem ipsum</p>
-              <h1 className="font-caption text-6xl md:text-8xl lg:text-[110px] text-brand-light-green">
+              <h1 className='font-caption text-6xl md:text-8xl lg:text-[110px] text-brand-light-green'>
                 Color Compass
               </h1>
-              <p className="w-2/3 md:w-[460px]">
+              <p className='w-2/3 md:w-[460px]'>
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                 Phasellus mattis tortor ut tortor cursus, ac maximus tortor.
               </p>
+              <a
+                href='#'
+                className='animate-pulse hover:[animation-play-state:paused]'
```

Comment:
I love the animation how still. I wonder could we use this to create a loading spinner too?

Reply:
Thank you. Of course we can use it if necessary.